tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs

```diff
@@ -81,12 +78,10 @@ public void FromNameGivenNameMediumShouldBeEqual()
         [Test]
         public void FromNameGivenNameExtraLargeShouldThrow()
         {
-            var expectedName = "extra large";
+            const string expectedName = "extra large";
```

Comment:
Should this have a capitalised first letter?

Reply:
As above.

Reply:
That's fair, approved